content/posts/macinplanck-configuration/index.md

```diff
@@ -0,0 +1,326 @@
+
+Some front-matter before I get into things:
+
+- I've been getting more and more into Vim, which is what inspired me to set up the keyboard the way that I did.
+- I use a Mac, so there are a bunch of mappings that I describe later that are only relevant for that OS. If you decide to duplicate this mapping, keep that in mind.
+- Currently, I've configured everything through [the Oryx configurator online](https://configure.ergodox-ez.com/); I know that more advanced macros and such exist in QMK, and that there are some things that I describe below that could be more efficiently accomplished via editing the `keymap.c` directly, I just didn't have the time (nor the patience) to get into that this time around.
+
+## Links and Resources
+
+Here are links to my current configuration:
+
+- [Planck EZ Configurator (my current, full setup)](https://configure.ergodox-ez.com/planck-ez/layouts/K0rgz/OvZ4W/0)
+- [Relevant `.bin`, source, and PDF files](https://github.com/macintacos/dotfiles/tree/master/keebs/macinplanck-ez) for my current layout.
+
+...and here are resources that I used to configure my setup:
+
+- [QMK Documentation](https://docs.qmk.fm/) (which is fantastic, btw) - a lot of acronyms below are coming straight out of this documentation, so it'd probably be handy to have this open as you read along.
+- [vimhelp.org](https://vimhelp.org/) (if you don't know Vim, one thing that you should know is that this is essentially your bible)
+- [r/olkb](https://www.reddit.com/r/olkb/) (solved pretty much any issue I encountered in this keymap build by searching here)
+- [Wally](https://github.com/zsa/wally/releases) (and it's `wally-cli` counterpart) for easily flashing new firmware.
+
+If [history is any indication](https://p-37FYgJ.b1.n0.cdn.getcloudapp.com/items/yAuLrDer/Image+2020-01-01+at+19.47.45.png?v=7038d4a6d045ee767c6e307b663cb190), the configuration that I'm about to describe will probably be out of date by this time tomorrow, but I feel that writing down my thought process helps me pick out the holes in my logic and give me ideas for future improvements.
+
+But anyway, who cares? Onto those sweet, juicy layers to make sense of wtf I even did.
+
+## Layers
+
+I currently have 9 layers (`Lower`, `Base`, `Raise`, `Adjust`, `NORMAL`, `VISUAL`, `NRMLSHFT`, `WINDOWZ`, and `LGUI_ALL`):
+
+![](https://p-37FYgJ.b1.n0.cdn.getcloudapp.com/items/E0uEA9B5/Image+2020-01-01+at+17.04.16.png?v=01081508789b9b85ad9ebb151b1e22af)
+
+Not the biggest of drawbacks, but for whatever reason you cannot edit the names if the first four tabs, or delete them. Removing the ability to delete them was understandable (otherwise it's kinda not a Planck), but the inability to rename them didn't really make sense to me. Oh well!
```

Comment:
> Not the biggest of drawbacks, but for whatever reason you cannot edit the names if the first four tabs, or delete them . . .

"edit the names _in_ the first four tabs," I suspect.

Reply:
fixed